test/studies/shootout/meteor/kbrady/meteor-parallel.chpl

```diff
@@ -300,9 +300,9 @@ proc fillContiguousSpace(board, indx) {
 proc firstEmptyCell(cell, in min) {
   while (min == cell[0] || min == cell[1] ||
          min == cell[2] || min == cell[3] ||
-         min == cell[4]) {
+         min == cell[4]) do
```

Comment:
Personally I prefer having brackets when the block enclosed spans multiple lines, even if it only encloses one outer statement.  I think it makes things clearer.  I wouldn't make you revert this commit unless others in the group agreed, though.